Makefile

```diff
@@ -70,11 +77,14 @@ endif
 ifeq ($(BASE_OS),clearlinux)
   BASE_OS_TAG=$(BASE_OS_TAG_CLEARLINUX)
 endif
-
-# Option to Override release image.
-# Release image OS *must have* glibc version >= glibc version on BASE_OS:
-DIST_OS ?= $(BASE_OS)
-DIST_OS_TAG ?= $(BASE_OS_TAG)
+ifeq ($(BASE_OS),redhat)
+  BASE_OS_TAG=$(BASE_OS_TAG_REDHAT)
+  RELEASE_BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.2
```

Comment:
I think it's unused. In release image we use ubi-minimal and it's hardcoded in release Dockerfile.